### PR TITLE
Add github templates

### PR DESCRIPTION
**Pull request by tomdee**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+3.
+4.
+
+## Context
+<!--- How has this issue affected you? What are you trying to accomplish? -->
+<!--- Providing context helps us come up with a solution that is most useful in the real world -->
+
+## Your Environment
+<!--- Include as many relevant details about the environment you experienced the bug in -->
+* CNI plugin version:
```
Comment (caseydavenport) [inline .github/ISSUE_TEMPLATE.md]: This should probably say how to figure that out - people get confused about versions.

e.g:

```
* CNI plugin version (e.g. `/opt/cni/bin/calico --version`)
```

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+* Orchestrator version (e.g. kubernetes, mesos, rkt):
```
Comment (caseydavenport) [inline .github/ISSUE_TEMPLATE.md]: Maybe "Orchestrator and version".

We want to know both what orchestrator it is, and the version of the orchestrator they're using.

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+* CNI config:
+* Operating System and version:
```
Comment (caseydavenport) [inline .github/ISSUE_TEMPLATE.md]: Again, is there a specific command we want for this one?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+## Description
+A few sentences describing the overall goals of the pull request's commits. 
+Please include
```
Comment (caseydavenport) [inline .github/PULL_REQUEST_TEMPLATE.md]: This should also include links to any issues that this PR fixes.

File: .github/ISSUE_TEMPLATE.md (hunk 3 above)
Comment (tomdee) [inline .github/ISSUE_TEMPLATE.md]: I can't think of one

**Pull request merged**